### ISSUE TITLE
loss=nan

### ISSUE THREAD

**Issue by jiangbaobao** [TimDettmers/bitsandbytes#646]
Python 3.10.8
pip 23.2.1 from D:\kohya_ss\venv\lib\site-packages\pip (python 3.10)

A matching Triton is not available, some optimizations will not be enabled.
Error caught was: No module named 'triton'

accelerate launch --num_cpu_threads_per_process=2 "./train_network.py" --pretrained_model_name_or_path="D:/lora-scripts/sd-models/model.ckpt" --train_data_dir="E:\Lora\input" --resolution="512,832"
                         --output_dir="E:\Lora\output" --logging_dir="E:\Lora\log" --network_alpha="64" --save_model_as=safetensors --network_module=networks.lora --text_encoder_lr=5e-05 --unet_lr=0.0001 --network_dim=64
                         --output_name="celine" --lr_scheduler_num_cycles="12" --no_half_vae --learning_rate="0.0001" --lr_scheduler="constant" --train_batch_size="1" --max_train_steps="68640" --save_every_n_epochs="2"
                         --mixed_precision="fp16" --save_precision="fp16" --seed="1334" --caption_extension=".txt" --cache_latents --optimizer_type="AdamW8bit" --max_data_loader_n_workers="1" --clip_skip=2 --bucket_reso_steps=64    
                         --mem_eff_attn --xformers --bucket_no_upscale --noise_offset=0.05
![Uploading 微信截图_20230727162501.png…]()
